spec/classes/plugin/dns_rfc2136_spec.rb

```diff
@@ -55,7 +56,7 @@ class { 'letsencrypt':
             is_expected.to contain_file("#{pathprefix}/etc/letsencrypt/dns-rfc2136.ini").
               with_ensure('file').
               with_owner('root').
-              with_group('root').
+              with_group("#{rootgroup}").
```

Comment:
you can directly use the variable
```suggestion
              with_group(rootgroup).
```